web/models/auth_handler.ex

```diff
@@ -6,10 +6,12 @@ defmodule EspiDni.AuthHandler do
   alias Ueberauth.Auth
   alias EspiDni.TeamFromAuth
   alias EspiDni.UserFromAuth
+  import EspiDni.Gettext
 
   def init_from_auth(%Auth{} = auth) do
     with {:ok, team} <- TeamFromAuth.find_or_create(auth),
-         {:ok, user} <- UserFromAuth.find_or_create(auth, team) do
+         {:ok, user} <- UserFromAuth.find_or_create(auth, team),
+         {:ok, user} <- send_welcome_message(user) do
       {:ok, team, user}
     end
     |> case do
```

Comment:
Version `1.3` of elixir provides a nicer way of doing this: http://tuvistavie.com/2016/elixir-1-3/#with-on-steroids

But for `1.2` this is the (ugly) syntax